### PR TITLE
Added dcos-aws-cli.

### PR DESCRIPTION
**Pull request by joerg84**

### CODE REVIEW

**Comment by joerg84** [comment]
Example PR https://github.com/dcos/examples/pull/194

File: repo/packages/D/dcos-aws-cli/0/package.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "packagingVersion": "3.0",
```
Comment (ryadav88) [inline repo/packages/D/dcos-aws-cli/0/package.json]: change to 4.0

File: repo/packages/D/dcos-aws-cli/0/package.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  "name": "dcos-aws-cli",
+  "version": "1.0",
+  "maintainer": "joerg@mesosphere.io",
+  "minDcosReleaseVersion": "1.9",
+  "description": "DC/OS AWS CLI",
```
Comment (ryadav88) [inline repo/packages/D/dcos-aws-cli/0/package.json]: Add `The DC/OS AWS CLI simplifies some operations for cluster running on AWS. At this point, it only supports one command - listing public IPs.`

Comment (ryadav88) [inline repo/packages/D/dcos-aws-cli/0/package.json]: Add `preInstallNotes` and add this line to it: `This DC/OS Service is currently in preview. There may be bugs, incomplete features, incorrect documentation, or other discrepancies. Preview packages should never be used in production!`

Comment (ryadav88) [inline repo/packages/D/dcos-aws-cli/0/package.json]: Will this work with Enterprise DC/OS. If not, please make a note here.

File: repo/packages/D/dcos-aws-cli/0/resource.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{
+  "assets": {},
```
Comment (ryadav88) [inline repo/packages/D/dcos-aws-cli/0/resource.json]: why do you need the `assets` property

File: repo/packages/D/dcos-aws-cli/0/package.json (hunk 2 above)
Comment (joerg84) [inline repo/packages/D/dcos-aws-cli/0/package.json]: It does.

**Pull request merged**